src/js/types/beacon.ts

```diff
@@ -162,6 +162,39 @@ export interface BeaconServiceInfo {
   info?: GenericInfoField;
 }
 
+// ----------------------------
+// filters
+// ----------------------------
+
+// https://docs.genomebeacons.org/filters/
+export interface BeaconFilteringTermsResponse {
+  meta?: {
+    beaconId: string;
+  };
+  response?: {
+    filteringTerms: BeaconFilteringTermFromResponse[];
+  };
+}
+
+export interface BeaconFilteringTermFromResponse {
```

Comment:
Maybe just BeaconFilteringTerm?

Reply:
If this is a comment on the naming I agree it's not great. But, owing to complications in the beacon spec, I need to distinguish between:
- "filtering terms" in the response from beacon endpoint `/filtering_terms`
- "filtering terms" in the beacon query

These are related but do not match exactly. But I'll look at these again and see if I can come up with something cleaner.

Reply:
Kept types as they are but tried to making the naming a bit more clear